### ISSUE TITLE
Missing tags & headers section from traits

### ISSUE THREAD

**Issue by ITman1** [asyncapi/asyncapi-react#179]
**Description**

I tried to integrate this component into my project and I hit on two things which I miss and on https://playground.asyncapi.io/ they work. That is why I would to ask if there is any plan to support them also in this React component? Thank you.

**Attachments**

![image](https://user-images.githubusercontent.com/1372054/98954287-eeffcf00-24fd-11eb-8303-b16767b99007.png)

**Comment by derberg** [comment]
@ITman1 Hey. So you mention 2 different features here:
- tags are not supported but definitely should be, there is a list of not supported props https://github.com/asyncapi/asyncapi-react/issues/63 Would be great to get some help here
- traits work properly, at least to me. Traits are resolved by the parser that we use in the react component. So it is not about headers in traits, they work the same if they are in traits or not. I checked and looks like headers are properly supported 👇 
![Screenshot 2020-11-13 at 11 50 56](https://user-images.githubusercontent.com/6995927/99064951-f62ee780-25a6-11eb-83e8-aeec2e0c9317.png)

**Comment by ITman1** [comment]
Sorry, my bad, I have found a reason why it did not work for me. I tried to fix missing tag feature by putting ` [KAFKA]` suffix into channel name which resulted into invalid schema. This caused that traits stopped to work - even for the messages.

Today I also found that `required` fields are not visualised anyhow. We could maybe track it somewhere. Anyway thank you for prompt answer and closing this issue.
